src/blockdata/script.rs

```diff
@@ -53,6 +54,21 @@ use crate::schnorr::{TapTweak, TweakedPublicKey, UntweakedPublicKey};
 #[derive(Clone, Default, PartialOrd, Ord, PartialEq, Eq, Hash)]
 pub struct Script(Box<[u8]>);
 
+/// Bitcoin script (using slice reference internally).
+///
+/// This type is equivalent to [`Script`] but uses a borrowed array on the stack instead of heap
```

Comment:
It doesn't have to be on stack. Can be borrowed from anywhere.

Reply:
Ah yes, cheers.